src/Button.js

```diff
@@ -97,7 +97,7 @@ Button.defaultProps = {
  * state
  */
 Button.propTypes = {
-    children: propTypes.string,
+    children: propTypes.node,
```

Comment:
For `<DropdownButton>`, which renders more than just strings in the button (also an icon).